LinearRegression/LinearRegressionWPF/ViewModels/LinearRegressionViewModel.cs

```diff
@@ -0,0 +1,327 @@
+﻿using System;
```

Comment:
Please use the latest version of the MainViewModel in the main branch, then rename it to LinearRegressionViewModel.